nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -161,8 +161,6 @@ enum nrf_wifi_sys_commands {
 	NRF_WIFI_CMD_RAW_CONFIG_FILTER,
 	/** Command to configure packet injector mode or Raw Tx mode */
 	NRF_WIFI_CMD_RAW_TX_PKT,
-	/** Command to get extra RPU debug statistics */ 
-	NRF_WIFI_CMD_GET_DEBUG_STATS,
```

Comment:
which part of the commit log this belongs to? It's unclear from the description.

Reply:
[SHEL-2398]: Add lockup recovery stats event from UMAC.
According to the updated requirement, lockup recovery statistics have now been incorporated into "struct rpu_lmac_stats" instead of being handled as separate commands and events as before.

Reply:
Then I would suggest the removal be a separate commit explaining the same.